spec.md

```diff
@@ -677,12 +677,12 @@ message ValidateVolumeCapabilitiesRequest {
   Version version = 1;
 
   // The information about the volume to check. This is a REQUIRED
-  // field. 
+  // field.
   VolumeInfo volume_info = 2;
 
   // The capabilities that the CO wants to check for the volume. This
   // call SHALL return “supported” only if all the volume capabilities
-  // specified below are supported.
+  // specified below are supported. This field is REQUIRED.
```

Comment:
Without this change, is there value in calling this RPC? For example, if a CO wants to validate the existence of a volume, is it valid to invoke `ValidateVolumeCapabilities` with an empty capabilities list? If so, we should document that expectation. Otherwise mark this as `REQUIRED` as suggested here. @jieyu @saad-ali thoughts?